spyderlib/config/main.py

```diff
@@ -553,6 +553,7 @@ def is_ubuntu():
               'editor/show/hide outline': "Ctrl+Alt+O",
               'editor/show/hide project explorer': "Ctrl+Alt+P",
               'editor/new file': "Ctrl+N",
+              'Editor/Open last closed':"Ctrl+Shift+N",

```

Comment:
Ok, so issue #2415 asked for a new shortcut. So let's use the same one as Firefox: `Ctrl+Shift+T`